packages/desktop/components/popups/TokenInformationPopup.svelte

```diff
@@ -88,12 +87,7 @@
             {/if}
         </div>
 
-        <div class="space-y-3 flex flex-col items-center justify-center">
-            <AssetIcon {asset} chainId={asset.chainId} large />
-            <Text type={TextType.h2} fontWeight={FontWeight.bold}>
-                {getUnitFromTokenMetadata(asset.metadata)}
-            </Text>
-        </div>
+        <AssetTile classes="mb-2 pointer-events-none" onClick={() => {}} {asset} />
```

Comment:
We could remove the `mb-2`
```suggestion
        <AssetTile classes="pointer-events-none" onClick={() => {}} {asset} />
```